docs/auth/phone-auth.md

```diff
@@ -179,16 +189,16 @@ export default function PhoneVerification() {
   }
 
   // Handle the verify phone button press
-  async function verifyPhoneNumber(phoneNumber) {
-    const confirmation = await auth().verifyPhoneNumber(phoneNumber);
+  async function handlePhoneNumberVerification(phoneNumber) {
+    const confirmation = await verifyPhoneNumber(auth, phoneNumber);
     setConfirm(confirmation);
   }
 
   // Handle confirm code button press
   async function confirmCode() {
     try {
-      const credential = auth.PhoneAuthProvider.credential(confirm.verificationId, code);
-      let userData = await auth().currentUser.linkWithCredential(credential);
+      const credential = PhoneAuthProvider.credential(confirm.verificationId, code);
+      let userData = await auth.currentUser.linkWithCredential(credential);
```

Comment:
as mentioned above - `auth.currentUser` seems to be gone in modular - I believe it should be `credential.user` here ?